orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/FindImageFromClusterTask.groovy

```diff
@@ -119,6 +119,29 @@ class FindImageFromClusterTask extends AbstractCloudProviderAwareTask implements
     Set<String> imageNames = []
     Map<Location, String> imageIds = [:]
 
+    // Supplement config with regions from subsequent deploy steps:
+    def deployRegions = stage.execution.stages.findAll {
```

Comment:
Believe there should be a better way / helper method to only look at downstream stages.

Reply:
Took it from https://github.com/spinnaker/orca/blob/master/orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy#L80

But also saw https://github.com/spinnaker/orca/blob/master/orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java#L55 which I can use. Will poke to see if there is anything better, let me know if you recall of the top of your head

Reply:
Fixed it up, let me know what you think!